hack/license-header-check.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
```

Comment:
Can you please add a license header to this file?

Reply:
yes of course. added